modules/srv6/gr_srv6.h

```diff
@@ -0,0 +1,63 @@
+// SPDX-License-Identifier: BSD-3-Clause
+// Copyright (c) 2025 Olivier Gournet
+
+#ifndef _GR_DATAPATH_SRV6
+#define _GR_DATAPATH_SRV6
+
+#include "gr_srv6_api.h"
+
+#include <rte_ip6.h>
+
+#include <endian.h>
+#include <netinet/ip6.h>
+
+//
+// srv6 pkt header definition
+//
+#define IP6_ROUTING_HEADER_TYPE_SRV6 4
```

Comment:
I am not sure which constant this is referring to, but there already is another one in DPDK with the same value:

https://github.com/DPDK/dpdk/blob/fd233ad17e5ffa42d50f1625165a7fce3f1cbc5f/lib/net/rte_ip6.h#L505-L506